src/style.css

```diff
@@ -1 +1,80 @@
-/* Styles go here */
+body {
+  font-family: "Robota", sans-serif;
+}
+
+*{
+  margin: 0;
```

Comment:
there should be only `box-sizing`
reset `margin` and `padding` for tags you need